plugins/lime-plugin-locate/src/locateReducer.js

```diff
@@ -23,6 +23,12 @@ export const initialState = {
 export const reducer = (state = initialState, { type, payload, meta }) => {
 	switch (type) {
 
+		case LOCATION_CHANGE:
+			return Object.assign({}, state, {submitting: true, editting: false})
```

Comment:
Here you mention the editting state but you don't define it in the initialState. Is it ok?

Reply:
good catch, it works anyway but I have added it to be more clear about the initial state.